lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -590,7 +590,7 @@ public function computeChangeSet(ClassMetadata $class, $entity)
 
             foreach ($actualData as $propName => $actualValue) {
                 // skip field, its a partially omitted one!
-                if (! (isset($originalData[$propName]) || array_key_exists($propName, $originalData))) {
+                if (! array_key_exists($propName, $originalData)) {
```

Comment:
This is done for performance reasons (fcall is much slower than isset)

Reply:
Ah :disappointed: 

Is something that we can measure at userland to see if this patch affects it, or should we leave as it is? :smile: